plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -560,10 +560,9 @@ def _sendNextGcodeLine(self):
             line = line[:line.find(";")]
         line = line.strip()
 
-        # Don't send empty lines
+        # Don't send empty lines. But we do have to send something, so send m105 instead.
         if line == "":
-            self._gcode_position += 1
-            return
+            line = "M105"
```

Comment:
Instead of sending an M105, which also causes a response to be sent, couldn't you send an empty comment?
line = ";"